tempo/serve/constants.py

```diff
@@ -8,7 +10,13 @@
 DefaultRemoteFilename = "remote.pickle"
 DefaultEnvFilename = "environment.tar.gz"
 
-MLServerEnvDeps = ["mlserver==0.4.0"]
+MLServerEnvDeps = ["mlserver==0.4.1.dev1"]
```

Comment:
What are the changes in MLServer to require the dev1 release?

Reply:
https://github.com/SeldonIO/MLServer/pull/273 and https://github.com/SeldonIO/MLServer/pull/286

These are specific changes required for being able to serve pytorch models (via mlflow) and it needed numpy fixes.

Reply:
Now that the parallel inference fix for the outlier example is also in, should we build a `0.4.2` release and use that instead?